README.md

```diff
@@ -720,6 +720,47 @@ by `graphile-migrate watch` is defined. By default this is in the
 `migrations/current.sql` file, but it might be `migrations/current/*.sql` if
 you're using folder mode.
 
+#### Including external files in the current migration
+You can include external files in your `current.sql` to better assist in source
+control. These includes are always relative to the root of where graphile-migrate
+is running.
```

Comment:
@benjie Do we want to roll with `fixtures` or do we want to use something more generic like `entities` ?  I was asking chatgpt last night if there was a more common term for these sorts of things and `entities` is the best thing it could come up with.